flambe/compile/serialization.py

```diff
@@ -393,7 +414,11 @@ def load_state_from_file(path: str,
                         local_metadata[FLAMBE_STASH_KEY] = stash
                     full_prefix = prefix + STATE_DICT_DELIMETER if prefix != '' else prefix
                     _prefix_keys(component_state, full_prefix)
+                    _prefix_keys(component_state._metadata, full_prefix)
```

Comment:
I had issues loading this model http://10.150.0.47:49558/ due to an incompatibility issue. I fixed it by only using the _metadata attr if it exists.

```
                    if hasattr(component_state, '_metadata'):
                        _prefix_keys(component_state._metadata, full_prefix)
                        state.update(component_state)
                        # Load torch.nn.Module metadata
                        state._metadata.update(component_state._metadata)
```

Can we add this check to preserve retrocompatibility? This model was trained last week and it would become incompatible.

If not, at least can we provide an informative error message? Thank you :)

Reply:
Ok, this actually didn't fix the whole issue. Still debugging, since retraining would take at least 2 days.